content/docs/user-guide/dvc.md

```diff
@@ -147,3 +147,21 @@ metafile (`models/rf.mlem`) and `dvc.lock` only.
 
 Learn more about [DVC](https://dvc.org/doc) and how it can be useful for
 training your ML models.
+
+## Working with private repositories
+
+If you commit model metafiles to a private repo and use DVC to store binaries,
+you'll need to authorize both via SSH (for DVC) and via HTTPS (for MLEM).
```

Comment:
authorize -> authenticate
Would be more accurate everywhere here imo

Reply:
Fixed.